### PR TITLE
HS-55 choose the role screen UI

### PR DESCRIPTION
**Pull request by asoi1990**

### CODE REVIEW

**Review by hellomentorplus — CHANGES_REQUESTED** [review]

**Comment by suesitran** [comment]
@asoi1990 please merge from dev and try again

File: lib/features/sign_up/view/choosing_roles_view.dart [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            icon: const Icon(Icons.arrow_back, color: Colors.black),
+            onPressed: () => {},
+          ),
+          bottom: PreferredSize(
+            preferredSize: Size(size.width, 0),
+            child: const LinearProgressIndicator(
+              backgroundColor: HSColor.neutral2,
+              color: HSColor.neutral6,
+              value: 0.75,
+              semanticsLabel: 'Linear progress indicator',
```
Comment (suesitran) [inline lib/features/sign_up/view/choosing_roles_view.dart:34]: I think we missed accessibility requirements in the story, thanks for adding it, but please move it to localisation

File: lib/features/sign_up/view/choosing_roles_view.dart [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+          child: Column(
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              children: [
+                Column(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    Text(
+                      HatSpaceStrings.of(context).chooseUserRole.toString(),
```
Comment (suesitran) [inline lib/features/sign_up/view/choosing_roles_view.dart:51]: U don't need to use toString() here

File: lib/features/sign_up/view/choosing_roles_view.dart [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+                      style: Theme.of(context).textTheme.displayMedium,
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.only(top: 16),
+                      child: Text(
+                        HatSpaceStrings.of(context)
+                            .chooseUserRoleDescription
+                            .toString(),
```
Comment (suesitran) [inline lib/features/sign_up/view/choosing_roles_view.dart:59]: U don't need to use toString() here

File: lib/features/sign_up/view/choosing_roles_view.dart [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+                                    : totalNumberOfCheckedTextBoxes--;
+                              });
+                            },
+                          );
+                        }),
+                  ],
+                ),
+                Expanded(child: Container()),
+                PrimaryButton(
+                  label: HatSpaceStrings.of(context).continueBtn.toString(),
```
Comment (suesitran) [inline lib/features/sign_up/view/choosing_roles_view.dart:84]: U don't need to use toString() here

File: lib/features/sign_up/view/choosing_roles_view.dart [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+                  onPressed: totalNumberOfCheckedTextBoxes > 0 ? () {} : null,
+                ),
+                Padding(
+                  padding: const EdgeInsets.only(top: 33),
+                  child: TextOnlyButton(
+                    label: HatSpaceStrings.of(context).cancelBtn.toString(),
```
Comment (suesitran) [inline lib/features/sign_up/view/choosing_roles_view.dart:90]: U don't need to use toString() here

**Review by suesitran — COMMENTED** [review]
Please fix my comments above

**Review by suesitran — APPROVED** [review]

**Review by hellomentorplus — APPROVED** [review]

**Pull request merged**